src/Mod/Spreadsheet/Gui/AppSpreadsheetGui.cpp

```diff
@@ -52,52 +52,51 @@ void loadSpreadsheetResource()
 }
 
 namespace SpreadsheetGui {
-class Module : public Py::ExtensionModule<Module>
-{
-public:
-    Module() : Py::ExtensionModule<Module>("SpreadsheetGui")
+    class Module : public Py::ExtensionModule<Module>

```

Comment:
It looks like an auto-formatter messed some things up here -- when I merge it I'm going to clean this up so we can minimize the diff (though some of the original formatting was pretty janky too, so  I'll probably fix that).

Reply:
Eh, never mind, it doesn't really minimize the diff once you start fixing the blatant problems with the original formatting. Merging as-is.